### PR TITLE
feat(LambdaGeneration): add presence

### PR DESCRIPTION
**Pull request by Valley152**
## Description 

Adds a presence for LambdaGeneration.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image

image

image

</details>

### CODE REVIEW

File: websites/L/LambdaGeneration/presence.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+				}
+				pagee[0] += ":";
+				pagee[1] = splitted[0];
+				break;
+			}
+		}
+	}
+
+	const presenceData: PresenceData = {
+		largeImageKey: "lambda",
```
Comment (theusaf) [inline websites/L/LambdaGeneration/presence.ts:89]: Imgur links are preferred over Discord asset names.
```suggestion
		largeImageKey: "https://i.imgur.com/XSPtxro.png",
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Comment by Valley152** [comment]
What are these errors with "chalk" and all the different colours?

File: websites/L/LambdaGeneration/presence.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+			["valve"]: "Valve",
+			["lambdageneration"]: "LambdaGeneration",
+			["black-mesa"]: "Black Mesa",
+			["sfm"]: "Source Film Maker",
+			["sven-co-op"]: "Sven Co-op",
+			["gmod"]: "Garry's Mod",
+			["csgo"]: "CS:GO",
+			["l4d"]: "Left 4 Dead",
+		},
+		pagee: string[] = ["Viewing", "trending"];
```
Comment (EncryptedDev) [inline websites/L/LambdaGeneration/presence.ts:20]: I'm confused, why `pagee` not `page` or `pages`

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/L/LambdaGeneration/presence.ts (hunk 2 above)
Comment (Valley152) [reply websites/L/LambdaGeneration/presence.ts:20]: I thought `page` and `pages` was reserved

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**